### PR TITLE
Documentation Fix for Upload Status

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

Adding additional context for upload status options

## Why?

NAC-1553

## This PR has:

- [ ] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [x] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [ ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

**Review by bgfoshay — APPROVED** [review]
👍 Looks good to me but maybe double-check with Skippy to make sure he's good

File: docs/openapi/generated/bulk-api/openapi.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ Status Code **200**
 |»» participants_uploaded|integer|false|none|The number or participants uploaded into the NAC|
 |»» error_message|string|false|none|Error details as to why an upload failed|
 |»» completed_at|string|false|none|The timestamp when the requested upload was completed.|
-|»» status|string|true|none|The Upload's status|
+|»» status|string|true|none|COMPLETE - when successful<br> FAILED - The upload was unsuccessful. The NAC will continue to match against records of active participants from your previous successful upload<br> UPLOADING - when in the process of uploading|
```
Comment (cjhskippy) [inline docs/openapi/generated/bulk-api/openapi.md:209]: Can we clarify that "UPLOADING" as a status includes file processing and validation, not just the upload time? I can imagine a State recognizing that a file transfer has completed, getting a status of "UPLOADING", and contacting us to report an error, even though in this situation, everything would probably be working correctly.

How would you feel about this variation?
UPLOADING - The file is uploading to the NAC, or the file has uploaded but is still being validated

File: docs/openapi/generated/bulk-api/openapi.yaml [hunk 2]
```diff
@@ -185,7 +185,10 @@ paths:
                         example: '2022-07-11T19:54:28.303961Z'
                       status:
                         type: string
-                        description: The Upload's status
+                        description: |-
+                          COMPLETE - when successful
+                           FAILED - The upload was unsuccessful. The NAC will continue to match against records of active participants from your previous successful upload
+                           UPLOADING - when in the process of uploading
```
Comment (cjhskippy) [inline docs/openapi/generated/bulk-api/openapi.yaml:191]: Same note about uploading as above

File: etl/docs/openapi/getUploadById.yaml [hunk 3]
```diff
@@ -57,7 +57,7 @@ get:
                     example: "2022-07-11T19:54:28.303961Z"
                   status:
                     type: string
-                    description: "The Upload's status"
+                    description: "COMPLETE - when successful\n FAILED - The upload was unsuccessful. The NAC will continue to match against records of active participants from your previous successful upload\n UPLOADING - when in the process of uploading"
```
Comment (cjhskippy) [inline etl/docs/openapi/getUploadById.yaml:60]: Same note about uploading status as above (sorry if repeating the comments isn't helpful)

**Review by cjhskippy — COMMENTED** [review]
Not a hard blocker, but I suggest a little clarification on the "UPLOADING" status to prevent States from contacting us for help if they get that status after a file transfer has completed.  It's just a minor clarification to ensure users know that "UPLOADING" status also includes the time we spend on validation.

**Pull request merged**